model/commitqueue/commit_queue.go

```diff
@@ -212,13 +214,13 @@ func (cq *CommitQueue) RemoveItemAndPreventMerge(issue, patchType string, versio
 }
 
 func preventMergeForItem(patchType string, versionExists bool, item CommitQueueItem, user string) error {
-	if patchType == PRPatchType && item.Version != "" {
+	if patchType == SourcePullRequest && item.Version != "" {
 		if err := clearVersionPatchSubscriber(item.Version, event.GithubMergeSubscriberType); err != nil {
 			return errors.Wrap(err, "can't clear subscriptions")
 		}
 	}
 
-	if patchType == CLIPatchType && versionExists {
+	if patchType == SourceCommandLine && versionExists {
```

Comment:
Now that we're adding new items as SourceDiff, these lines should check SourceCommandLine and SourceDiff, right?

Reply:
No, what's passed in to here is still the setting from the project ref. This will change in the next pr

Reply:
There are a few places we're using SourceDiff now though -- restartClIItems, the enqueue rest route, the enqueue resolver. Should those still use SourceCommandLine until the next PR then?

Reply:
those places are writing it to the commit queue item, which isn't used anywhere yet. It's just so that when we next deploy the data will be correct to read from those instead of the project ref

Reply:
oh right! okay makes sense, thanks